rules/list-item.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const caseOf = require('case').of;
+const visit = require('unist-util-visit');
+
+module.exports = (ast, file, language, done) => {
+	visit(ast, 'list', list => {
+		let position;
+		for (const item of list.children) {
+			position = JSON.parse(JSON.stringify(item.position));
+			if (position.start.line !== position.end.line) {
+				position.start.line++;
+				file.warn('Items must start with \'- [name][link]\'', position);
+			}
+
+			try { // TODO: seems like unist-util-visit does not supports sublists
```

Comment:
Can you add a link to the issue https://github.com/wooorm/remark/issues/198 here?